javascript/ql/src/semmle/javascript/Concepts.qll

```diff
@@ -65,3 +65,27 @@ abstract class DatabaseAccess extends DataFlow::Node {
   /** Gets an argument to this database access that is interpreted as a query. */
   abstract DataFlow::Node getAQueryArgument();
 }
+
+/**
+ * A data flow node that reads persistent data.
+ */
+abstract class PersistentReadAccess extends DataFlow::Node {
+
+  /**
+   * Gets the corresponding persistent write, if any.
```

Comment:
s/the/a/, I assume? (I don't mind the occasional "the" where technically "a" would be correct, but I guess in this case it is very much the rule that there is more than one write for a read.)